### PR TITLE
Game can now only pick prison break event if people are actually in prison

### PR DESCRIPTION
**Pull request by ghost**
[tweak][tested]

## What this does
Does not fire this event if prison areas have no actual mobs in them that can access brig doors, also doubles chance of firing from old value to make it appear in this situation more.

## Why it's good
Frees up an event from an area not in use.

## Changelog
:cl:
 * tweak: Prison breaks will no longer happen if there are no signs of life within cells without access to open them, but are now twice as common when there is.

### CODE REVIEW

**Comment by Kurfursten** [comment]
An improvement to be sure but didn't go far enough. The event can still fire just from warden idling at desk 

It should be a living human who does not have access to the brig front doors.

**Comment by ghost** [comment]
> An improvement to be sure but didn't go far enough. The event can still fire just from warden idling at desk
> 
> It should be a living human who does not have access to the brig front doors.

Fixed

File: code/modules/events/prison_break.dm [hunk 1]
```diff
@@ -6,7 +6,28 @@
...
+	var/foundBasic = TRUE
+	for(var/area/A in areas)
+		if(istype(A, /area/security/prison) || istype(A, /area/security/brig))
+			prisonAreas += A
+			var/list/areaMobs = mobs_in_area(A)
+			if(areaMobs && areaMobs.len)
+				foundBasic = TRUE
+			for(var/mob/living/carbon/human/H in areaMobs)
+				var/list/access = H.GetAccess()
+				if(!(access_brig in access))
```
Comment (Kurfursten) [inline code/modules/events/prison_break.dm:21]: unless you check if they're alive, you can still trigger greytide by storing a dead body in evidence storage (more common than you probably think)

Comment (ghost) [reply code/modules/events/prison_break.dm:21]: Fixed

**Pull request merged**